src/style.css

```diff
@@ -1,3 +1,85 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
+:root {
+  --blue-color: #00acdc;
+}
+
+html {
+  font-size: 12px;
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 50px;
+}
+
+.header__img {
+  width: 40px;
+  height: 40px;
+}
+
+.header__img img {
+  width: 100%;
+}
+
+.header__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.header__item {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 60px;
+  list-style: none;
```

Comment:
Avoid using tag names for styling. Here, `list-style: none;` can be applied to the `.header__list` class instead of the `.header__item` class, which represents a single item.